custom_components/tuya_local/devices/emos_ip300_camera.yaml

```diff
@@ -0,0 +1,434 @@
+name: Camera with light
+products:
+  - id: wauxpj9p977uxhvb
+    manufacturer: EMOS
+    model: IP-300
+entities:
+  - entity: camera
+    dps:
+      - id: 134
+        type: boolean
+        name: motion_enable
+        optional: true
+        mapping:
+          - dps_val: null
+            value: false
+            hidden: true
+      - id: 150
+        type: boolean
+        name: record
+        optional: true
+        mapping:
+          - dps_val: null
+            value: false
+            hidden: true
+      - id: 115
+        type: base64
+        optional: true
+        sensitive: true
+        name: snapshot
+      - id: 185
```

Comment:
May be better as a event. Log messages are related to a specific point in time, and are often of enough interest to have a entity dedicated to them.
Check how other cameras with dp id 185 handle them (particularly the more recently added ones).